gpflow/conditionals/util.py

```diff
@@ -469,7 +469,7 @@ def mix_latent_gp(W, g_mean, g_var, full_cov, full_output_cov):
     if not full_cov:
         shape_constraints.append((g_var, [..., "N", "L"]))
     else:
-        # TODO cannot assert g_var shape here because of the inner "leading"
+        # NOTE(awav) cannot assert g_var shape here because of the inner "leading"
```

Comment:
this already links to an issue - I would leave this as a todo.

Reply:
`TODO:` - intention to do something, because we have control over it. `NOTE:` - we have to live with it, even if this is for a short time.